docs/customization_guide/inference_protocols.md

```diff
@@ -114,26 +114,27 @@ These options can be used to configure the KeepAlive settings:
 
 For client-side documentation, see [Client-Side GRPC KeepAlive](https://github.com/triton-inference-server/client/blob/main/README.md#grpc-keepalive).
 
-#### Limit Endpoint Access (BETA)
+### Limit Endpoint Access (BETA)
 
-In some use cases, Triton users may want to restrict the access of the protocols on a given endpoint.
-For example, there can be need for two separate protocol groups that one exposes standard inference
-protocols for user access, while the other one exposes other extension protocols for administration
-usage and should not be accessible by non-admin user.
+Triton users may want to restrict access to protocols or APIs that are
+provided by the GRPC or HTTP endpoints of a server. For example, users
+can provide one set of access credentials for inference APIs and
+another for model control APIs such as model loading and unloading.
 
-The following option can be specified to declare an restricted protocol group:
+The following options can be specified to declare a restricted
+protocol group (GRPC) or restricted API group (HTTP):
 
 ```
 --grpc-restricted-protocol=<protocol_1>,<protocol_2>,...:<restricted-key>=<restricted-value>
+--http-restricted-api=<API_1>,API_2>,...:<restricted-key>=<restricted-value>
 ```
 
 The option can be specified multiple times to specifies multiple groups of
-protocols with different restriction settings.
+protocols or APIs with different restriction settings.
 
-* `protocols` : A comma-separated list of protocols to be included in this
-group. Note that currently a given protocol is not allowed to be included in
-multiple groups. The following protocols are currently recognized by all network
-protocol types mentioned above:
+* `protocols / APIs` : A comma-separated list of protocols / APIs to be included in this
```

Comment:
(another nit not for this release)

I think we touched on this briefly when implementing, but if this is still considered beta or subject to change, my 2 cents would be to try to unify naming in the next release or something. ex:

To restrict access to certain Triton APIs over the network...:
```
--grpc-restricted-protocol
--http-restricted-protocol
```

Rather than having to say `protocols / API` in every spot, and having different arg names for each. Upon reading it, my first question is, "what's the difference between a protocol and an API here?"

I think the cli args can say `protocol` just to keep the `grpc` version unchanged since it came first, but we can describe them as either protocol **or** API in the text around it and be consistent with it.

Reply:
sounds good - we can add a ticket to unify behavior / naming. Since its beta we can decide whether to keep the grpc handling as is or modify it. having `--http-restricted-protocol` seems odd as it is a protocol itself but maybe its the least bad unified name. could consider, `--http-restricted-feature`, `--grpc-restricted-feature`. perhaps ...